ansible/roles/common/tasks/main.yml

```diff
@@ -31,6 +31,12 @@
     bin_dir: "/opt/bin"
   when: ansible_distribution == "CoreOS"
 
+# On CentOS with kubernetes installed from packages binaries are in /usr/bin/
+- name: Set the bin directory path for CentOS
+  set_fact:
+    bin_dir: "/usr/bin"
+  when: ansible_distribution == "CentOS" and source_type == "packageManager"
+
```

Comment:
@rutsky this might not be the best place to fix bin_dir. @eparis can you confirm if you want the project to follow [1] for the default bin_dir?

[1] https://github.com/kubernetes/contrib/tree/master/init